.babelrc

```diff
@@ -7,7 +7,8 @@
 		} ]
 	],
 	"plugins": [
-		"transform-runtime"
+		"transform-runtime",
+		"transform-object-rest-spread"
```

Comment:
I agree, I also felt the need of Class Properties in my other PR, I don't know how you feel about adding this as well.